modules/ROOT/pages/tls-configuration.adoc

```diff
@@ -6,7 +6,7 @@ endif::[]
 
 TLS is a cryptographic protocol that provides communications security for your Mule app. TLS offers many different ways of exchanging keys for authentication, encrypting data, and guaranteeing message integrity. This topic describes TLS as supported by Mule and how to configure TLS in Mule apps.
 
-Mule runtime engine 4.x supports Transport Layer Security (TLS) 1.1 and 1.2.
+Mule runtime engine 4.x supports Transport Layer Security (TLS) 1.1 and 1.2. If Mule is running on a JDK that supports TLS 1.3, Mule can run applications that use TLS 1.3.
```

Comment:
We should first update https://docs.mulesoft.com/mule-runtime/4.4/tls-configuration because it has an asterisk pointing to nowhere. Asterisk can be a note for on premises or something like that saying that 1.3 support is JDK dependant but it's present in all the latest versions of our recommended versions: https://docs.mulesoft.com/mule-runtime/4.4/hardware-and-software-requirements#required-software. Then we can put exactly the same (including the compatibility matrix) here.

Reply:
I just added the table and removed the redundant sentence.